crates/starknet/Cargo.toml

```diff
@@ -11,6 +11,11 @@ blockifier = { workspace = true, features = ["testing"]}
 cairo-felt = { workspace = true }
 cairo-lang-starknet = { workspace = true }
 clap = { workspace = true }
+
+# TODO: perhaps not the best solution to have ethers inside Starknet crate.
+# Here for the MVP, to be checked with spaceshard team.
+ethers = { version = "2.0.8" }
```

Comment:
for me, it's ok since it's needed for l1l2, I would remove this todo, @FabijanC please also confirm that

Reply:
in old devnet we use web3 library

Reply:
Maybe @glihm was worried about the location of this dependency (i.e. it not being defined in the root Cargo.toml). If that is the concern, I summon @marioiordanov

Reply:
@FabijanC correct. But it looks like ok for now, let me know what do you think @marioiordanov.

Reply:
its ok as it is